firmware/thruna/DCM/Src/App/states/App_DriveState.c

```diff
@@ -67,6 +69,13 @@ static void DriveStateRunOnTick100Hz(struct StateMachine *const state_machine)
     {
         App_SetPeriodicCanSignals_TorqueRequests();
 
+        // regen_switch_enabled = ; TODO: what switch to tie this to?
```

Comment:
Right now the only choice here is the AuxSwitch (for Thruna at least), which I also use for torque vectoring right now, but I might not need it in the future once torque vectoring is the default drive mode. I used the AuxSwitch in my branch if you need to see how to do it

Reply:
ok i'll add that to aux switch but torque vectoring definitely needs a switch as we saw at comp